openstack/03a-IPHC/frag.c

```diff
@@ -436,7 +435,6 @@ static void store_fragment(OpenQueueEntry_t *msg, uint16_t size, uint16_t tag, u
     for (i = 0; i < FRAGMENT_BUFFER_SIZE; i++) {
         if (frag_vars.fragmentBuf[i].datagram_tag == tag && frag_vars.fragmentBuf[i].datagram_offset == offset) {
             openqueue_freePacketBuffer(msg);
-            return;
```

Comment:
Should this one be removed?  Without the return, the following code will be executed. Unless this is an error, I would recommend to reverse this change.